farming/logics/traits/getters.rs

```diff
@@ -0,0 +1,10 @@
+use crate::traits::farming::Data;
+use openbrush::traits::Storage;
+
+#[openbrush::trait_definition]
+pub trait FarmingGetters: Storage<Data> {
```

Comment:
So, we are going to add all public state variable getters (user info, etc.) into this trait, right?
I love this decomposition!

Reply:
Yes =)